setup.py

```diff
@@ -74,6 +74,7 @@ def read_file(name):
     'gunicorn==19.5.0',
     'html5lib==0.9999999',
     'influxdb==3.0.0',
+    'ipgetter==0.6',
```

Comment:
Yup. Telesom is asking for an IP in their API, so I implemented a remote IP getter, but then it turned out you should always return ":1"... :-o